packages/storybook-a11y-test/tests/index.test.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-env node */
+import path from 'path';
+
+import {A11yTestRunner} from '../src/index';
+
+const buildDir = path.join(__dirname, './fixtures/storybook');
+
+describe('can test a story', () => {
```

Comment:
Testing against fixtured data feels strange. It means we're testing against a specific point in time rather than using a storybook built with whatever version that is defined in the package.json.

I can see a point in time when the we update storybook in the dev dependencies, but don't touch the test fixtures and we think everything is fine because tests pass, but then when it comes to test the real latest storybook version everything breaks. I think that'd be suprising and it wouldn't be clear that you'd need to update the fixture data whenever you updated the storybook version.

I think running `yarn workspace @shopify/storybook-a11y-test run build-storybook --modern --output-dir=tests/fixtures/storybook` build in a `beforeAll` step here and testing against what that generates would be more bulletproof at the expense of taking a bit longer.


I also note that the creation of this fixture is not deterministic - I ran `yarn workspace @shopify/storybook-a11y-test run build-storybook --modern --output-dir=tests/fixtures/storybook` and it produced a bunch of different files compared to this output. It's not clear if anything meaningful actually changed, so keeping this fixture up to date has the potential to be churny.